packages/perseus/src/strings.ts

```diff
@@ -516,6 +517,10 @@ export type PerseusStrings = {
  * !! Note: Ensure that all escape sequences are double-escaped. (e.g. `\\text` -> `\\\\text`)
  */
 export const strings = {
+    characterCount: {
+        one: "%(used)s / %(num)s Character",
```

Comment:
Also here: a comment explaining the name `num` could be helpful for maintenance.